sensorimotor/sensorimotor/fast_general_temporal_memory.py

```diff
@@ -88,8 +88,7 @@ def burstColumns(self,
     activeCells = set()
     winnerCells = set()
     learningSegments = set()
-
-    unpredictedColumns = activeColumns - predictedColumns
+    unpredictedColumns = set(activeColumns) - predictedColumns
```

Comment:
I think when I was first writing this I was having some serialization issues. Shouldn't be a problem anymore though so I'll remove it.